content/en/docs/tasks/access-application-cluster/service-access-application-cluster.md

```diff
@@ -34,9 +34,14 @@ provides load balancing for an application that has two running instances.
 ## Creating a service for an application running in two pods
 
 1. Run a Hello World application in your cluster:
-   ```shell
-   kubectl run hello-world --replicas=2 --labels="run=load-balancer-example" --image=gcr.io/google-samples/node-hello:1.0  --port=8080
-   ```   
+    Here is the configuration file for the application Deployment:
+
+    {{< codenew file="service/access/hello-application.yaml" >}}
+
+    Create the application Deployment:
```

Comment:
Check the preview at https://deploy-preview-14980--kubernetes-io-master-staging.netlify.com/docs/tasks/access-application-cluster/service-access-application-cluster/.

Some ordinary text is getting formatted as code. You'll need to fiddle with indentation to fix this.